extensions/barrage/src/test/java/io/deephaven/extensions/barrage/chunk/BarrageColumnRoundTripTest.java

```diff
@@ -450,13 +450,19 @@ public void assertExpected(final WritableChunk<Attributes.Values> untypedOrigina
                         options, chunkType, type, fieldNodes.iterator(), bufferNodes.iterator(), dis);
                 Assert.eq(data.size(), "data.size()", rtData.size(), "rtData.size()");
                 validator.assertExpected(data, rtData, null);
+            } finally {
+                if (rtData != null) {
+                    rtData.close();
+                }
+                rtData = null;
             }
 
             // empty subset
             try (final BarrageProtoUtil.ExposedByteArrayOutputStream baos =
-                    new BarrageProtoUtil.ExposedByteArrayOutputStream()) {
-                final ChunkInputStreamGenerator.DrainableColumn column =
-                        generator.getInputStream(options, Index.CURRENT_FACTORY.getEmptyIndex());
+                    new BarrageProtoUtil.ExposedByteArrayOutputStream();
+                    final ChunkInputStreamGenerator.DrainableColumn column =
+                            generator.getInputStream(options, Index.CURRENT_FACTORY.getEmptyIndex());) {
```

Comment:
@nbauernfeind Is the Index going to be mutated, too? Should we be using it as a resource?

Reply:
That index is not mutated. It does get cloned by the DrainableColumn and that subset is closed when the column itself is closed.

We might need to tryWithResources it if empty indexes are no longer allowed to leak to GC.

Reply:
We're leaving it for the GC regardless of whether we close it. This is fine; I was only concerned about making sure we close it if we mutate it and then clone it in a more persistent structure.